src/service/dynamo.ts

```diff
@@ -134,6 +134,27 @@ class Dynamo implements DynamoService {
   constructor(url: string, authProvider?: () => Promise<string>) {
     this.url = url;
     this.authProvider = authProvider;
+
+    // Workaround to get a jump start on the API key creation (which can take ~30 seconds)
+    if (!this.url.startsWith("http://localhost")) {
+      this._fetch(`${this.url}/v1/graph/job/create`, { method: "GET" });
+    }
+  }
+
+  async retryFetch(input: RequestInfo, init?: RequestInit | undefined): Promise<Response> {
+    let response = await fetch(input, init);
+    const numRetries = 35;
+    let ii = 0;
+    while (response.status === 403 && ii < numRetries) {
+      try {
+        response = await fetch(input, init);
+        await new Promise((resolve) => setTimeout(resolve, 1000));
+        ii++;
+      } catch {
+        console.warn("Failed to fetch, retrying...");
```

Comment:
we need an ii++ inside the catch as well, or just move it from the existing one outside the try catch

Reply:
otherwise we could have an infinite loop